src/test/javascript/spec/component/exam/participate/summary/modeling-exam-summary.component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Course } from 'app/entities/course.model';
+import { ModelingExercise, UMLDiagramType } from 'app/entities/modeling-exercise.model';
+import { ModelingSubmission } from 'app/entities/modeling-submission.model';
+import { ModelingExamSummaryComponent } from 'app/exam/participate/summary/exercises/modeling-exam-summary/modeling-exam-summary.component';
+import { ModelingEditorComponent } from 'app/exercises/modeling/shared/modeling-editor.component';
+import * as chai from 'chai';
+import { MockComponent } from 'ng-mocks';
+import * as sinonChai from 'sinon-chai';
+
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe('ModelingExamSummaryComponent', () => {
+    let fixture: ComponentFixture<ModelingExamSummaryComponent>;
+    let comp: ModelingExamSummaryComponent;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [],
+            declarations: [ModelingExamSummaryComponent, MockComponent(ModelingEditorComponent)],
+            schemas: [],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(ModelingExamSummaryComponent);
+                comp = fixture.componentInstance;
+            });
+    });
+
+    it('should initialize', () => {
+        fixture.detectChanges();
+        expect(ModelingExamSummaryComponent).to.be.ok;
+    });
+
+    it('should show no submission when there is no uml model', () => {
+        comp.ngOnInit();
```

Comment:
Not a big deal but this is not necessary as detectChanges will call ngOnInit.

Reply:
Yeap, removed those.